src/cloud/project/project-conf-files_magento-app.md

```diff
@@ -5,507 +5,16 @@ functional_areas:
-   -  `soap`
-   -  `sockets`
-   -  `sodium`
-   -  `ssh2`
-   -  `sysvmsg`
-   -  `sysvsem`
-   -  `sysvshm`
-   -  `tidy`
-   -  `xdebug`
-   -  `xmlrpc`
-   -  `xsl`
-   -  `yaml`
-   -  `zip`
-   -  `pcntl`
-
-{:.bs-callout-warning}
-PHP compiled with debug is not supported and the Probe may conflict with XDebug or XHProf. Disable those extensions when enabling the Probe. The Probe conflicts with some PHP extensions like Pinba or IonCube.
-
-### Customize `php.ini` settings
-
-You can also create and push a `php.ini` file that is appended to the configuration maintained by {{site.data.var.ee}}.
-
-In your repository, the `php.ini` file should be added to the root of the application (the repository root).
-
-{:.bs-callout-info}
-Configuring PHP settings improperly can cause issues. We recommend only advanced administrators set these options.
-
-For example, if you need to increase the PHP memory limit:
-
-```bash
-memory_limit = 756M
-```
-
-For a list of recommended PHP configuration settings, see [Required PHP settings]({{ site.baseurl }}/guides/v2.3/install-gde/prereq/php-settings.html).
-
-After pushing your file, you can check that the custom PHP configuration has been added to your environment by [creating an SSH tunnel]({{ site.baseurl }}/cloud/env/environments-start.html#env-start-tunn) to your environment and entering:
-
-```bash
-cat /etc/php5/fpm/php.ini
-```
-
-{:.bs-callout-info}
-If you use {{site.data.var.mcd-prod}} for local development, see [Docker service containers]{{site.baseurl}}//cloud/docker/docker-containers-service.html#fpm-container) for information about using a custom `php.ini` file in a Docker environment.
-
-## Workers
-
-You can define zero or multiple work instances for each application. A worker
-instance runs as a container, independent from the web instance and without
-a running Nginx instance. Additionally, you do not need to set up a web server on
-the worker instance (using Node.js or Go) because the router cannot direct public
-requests to the worker.
-
-A worker instance has the exact same code and compilation output as a web instance.
-The container image is built once and deployed multiple times if needed using the
-same `build` hook and `dependencies`. You can customize the container and
-allocated resources.
-
-Use worker instances for background tasks including:
-
--  Tasks like queue workers or updating indexes.
--  Tasks to run periodic reporting that are too long for a cron job.
--  Tasks should happen "now", but not block a web request.
--  Tasks are large enough that they risk blocking a deploy, even if they are subdivided.
--  The task in question is a continually running process rather than a stream of discrete units of work.
+### [Properties]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app_properties.html)
 
-A basic, common worker configuration could look like this:
+### [Variables]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app_variables.html)
```

Comment:
With the above suggestion, you can remove all of these headings.

Reply:
Okay I will look into those. It will take time to rename all the files as there are links to them everywhere!

Reply:
Yes, sorry about that. Just don't want to continue that wonky file naming. I honestly have no idea what `project-conf-files_` meant. It's from some year ending in B.C. I think.

Reply:
Before Cloud?

Reply:
What I'll do it get the content right first (just in case we need new files) - then when you give a 'verbal' oaky - I'll refactor all the files name and then you can do a formal review - that okay?